src/main/java/ca/uhn/fhir/jpa/starter/dotBase/interceptors/AuthenticationInterceptor.java

```diff
@@ -53,7 +53,8 @@ private String getAuthenticatedUser(RequestDetails theRequestDetails) {
       Claims jwt = Authentication.verifyAndDecodeJWT(theRequestDetails);
       return getUsername(jwt);
     }
-    if (theRequestDetails.getHeader("X-Forwarded-User") != null) {
+    String xForwardedUser = theRequestDetails.getHeader("X-Forwarded-User");
+    if (xForwardedUser != null && !xForwardedUser.equals("")) {
       return theRequestDetails.getHeader("X-Forwarded-User");
```

Comment:
Return the already retrieved value instead of reusing the getter.
```suggestion
      return xForwardedUser;
```